### PR TITLE
Create convert-and-round-values.md

### PR DESCRIPTION
**Pull request by john-knowles**
Thank you @shawn149 for this one!
@adamkinniburgh does this ```{{Number(value/"something").toFixed("decimal")}}``` look ok?

### CODE REVIEW

**Comment by jellyash** [comment]
The screenshot is confusing as it shows the number, rather than {{value}} - I get why you did this to test it worked, but it's not the same as the custom label.

**Comment by jellyash** [comment]
Sorry, I hit Close instead of Commnet - Why are they so close together -.-

**Comment by john-knowles** [comment]
> The screenshot is confusing as it shows the number, rather than {{value}} - I get why you did this to test it worked, but it's not the same as the custom label.

Good spot, I've updated the image.

File: mustache-and-custom-labels/convert-and-round-values.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+This snippet of JavaScript can be used in the SquaredUp mustache and label fields when dealing with metrics.
+
+```{{Number(value/"something").toFixed("decimal")}}```
+
+## Working example: Converting bytes to gigabytes
+We're calling an API that responds with the total number amount of space on a disk as 221,300,711,424 Bytes, and want to show that in a Scalar tile as Gigabytes to two decimal places.
+
+Scalar Vaue formatter
+
+```{{Number(value/1073741824).toFixed(2)}}```
```
Comment (adamkinniburgh) [inline mustache-and-custom-labels/convert-and-round-values.md:14]: Perfect!

Sometimes it's useful to break out the equation just to make it more obvious what's going on i.e. `value/1024/1024/1024` but it works either way so I'll happily click the green button :)

**Review by adamkinniburgh — APPROVED** [review]

File: mustache-and-custom-labels/convert-and-round-values.md (hunk 1 above)
Comment (john-knowles) [reply mustache-and-custom-labels/convert-and-round-values.md:14]: What I was not sure about is using quotes " " to show that something can be changed
`{{Number(value/"something").toFixed("decimal")}}`

Comment (john-knowles) [reply mustache-and-custom-labels/convert-and-round-values.md]: Thank you - made that change.

**Review by jellyash — APPROVED** [review]

**Pull request merged**